deploy/ui/form.json

```diff
@@ -1245,6 +1245,55 @@
                     }
                   ]
                 },
+                {
+                  "label": "Enable Kafka Emitter",
+                  "type": "object",
+                  "max": 1,
+                  "jsonPath": "$.spec.objects.servers[*].kafkaJbpmEventEmitters",
+                  "visible": true,
+                  "fields": [
+                    {
+                      "label": "Bootstrap Servers",
+                      "type": "text",
+                      "required": false,
+                      "jsonPath": "$.spec.objects.servers[*].kafkaJbpmEventEmitters.bootstrapServers",
+                      "originalJsonPath": "$.spec.objects.servers[*].kafkaJbpmEventEmitters.bootstrapServers",
+                      "description": "A comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster"
```

Comment:
It sill not aligned with https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/550/files

Reply:
Updated